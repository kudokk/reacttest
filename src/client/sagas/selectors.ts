import { State } from '../reducers';

export const getOrgs = (state: State) => state.orgs;
