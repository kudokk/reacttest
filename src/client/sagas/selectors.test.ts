import { createStore } from 'redux';
import * as selectors from './selectors';
import { rootReducer } from '../reducers';

test('should get orgs state', () => {
  const storeState = createStore(rootReducer).getState();

  expect(selectors.getOrgs(storeState as any)).toMatchSnapshot();
});
