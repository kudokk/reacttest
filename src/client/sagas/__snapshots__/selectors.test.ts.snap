// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should get orgs state 1`] = `
Object {
  "isFetching": false,
  "name": "",
  "repos": Array [],
}
`;
